README.md

```diff
@@ -75,11 +75,48 @@ Image: .\BUILD\<TARGET_NAME>\ARM\mbed-os-example-bootloader.bin
 
 It creates two binary files. The original uncombined image is in the output directory: <project-name>_application.bin and the bootloader image is <project-name>.bin.
 
-## Next steps
-
 When the build succeeds, you have created a bootloader for your target. This bootloader defines the update file to be located at ``"/sd/mbed-os-example-bootloader-blinky_application.bin"``. This is the binary bootloader flashes and then jumps to.
 
-The next step is to build an application you can combine with your bootloader to create a loadable image. You can find a blinky application that uses this bootloader in the [mbed-os-example-bootloader-blinky](https://github.com/ARMmbed/mbed-os-example-bootloader-blinky) project.
+## Set up application to use bootloader
+
+The next step is to build an application you can combine with your bootloader to create a loadable image. 
+
+1. Update your board to use the newly created bootloader image. To do this, set the target (replace ``<TARGET_NAME>`` with your target name) value `bootloader_img` to the file path of the bootloader image.
+
+```
+    "target_overrides": {
+        ...
+        "<TARGET_NAME>": {
+            "target.bootloader_img": "bootloader/<TARGET_NAME>.bin"
+        },
+        ...
+```
+
+2. Invoke `mbed compile`, and specify the name of your platform and your favorite toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the ARM Compiler 5:
```

Comment:
Query: Is "favorite" toolchain correct? What if my favorite toolchain isn't the one I'm using? Could we change "and your favorite toolchain" to "and the toolchain you're using"?